.eslintrc-frontend.yml

```diff
@@ -26,3 +28,9 @@ rules:
   react/jsx-sort-props: 'error'
   react-hooks/rules-of-hooks: 'error'
   react-hooks/exhaustive-deps: 'error'
+
+  '@typescript-eslint/no-unused-vars': ['error', { 'args': 'none' }]
+
+  # Argh.
```

Comment:
:rofl:

Reply:
Comment duly copied and pasted from
some other project.